src/components/sidenav/sidenav.html

```diff
@@ -0,0 +1,14 @@
+<div class="md-sidenav-backdrop"
+     (click)="(start_ && (start_.mode == 'side' || start_.close()));
```

Comment:
These expressions are way too long to be in the template; they should be methods on the component instance.